src/frontend/src/apis/finance.api.ts

```diff
@@ -82,8 +83,10 @@ export const deleteReimbursementRequest = (id: string) => {
  */
 export const getAllExpenseTypes = () => {
   return axios.get(apiUrls.getAllExpenseTypes(), {
-    transformResponse: (data) =>
-      JSON.parse(data).map((expenseType: any) => ({ ...expenseType, id: expenseType.expenseTypeId }))
+    transformResponse: (data) => {
+      const parsedData = JSON.parse(data) as ExpenseType[];
+      return parsedData.map((expenseType) => ({ ...expenseType, id: expenseType.expenseTypeId }));
```

Comment:
Not sure why we even had this code setting the id like this, you can condense the method to just return JSON.parse(data) as ExpenseType[]

Reply:
To be honest I wasn't so sure but this was the previous code

```ts
    transformResponse: (data) =>
      JSON.parse(data).map((expenseType: any) => ({ ...expenseType, id: expenseType.expenseTypeId }))
```

does id have to be set to expenseTypeId?

Reply:
I just changed it